runners/core-construction-java/src/main/java/org/apache/beam/runners/core/construction/External.java

```diff
@@ -194,6 +195,7 @@ public OutputT expand(InputT input) {
                     PValues.expandInput(PBegin.in(p)),
                     ImmutableMap.of(entry.getKey(), (PCollection<?>) entry.getValue()),
                     Impulse.create(),
+                    ResourceHints.create(),
```

Comment:
Maybe drop a TODO here about figuring out what to do with resource hints across multi-language pipelines?

Reply:
Done